modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Providing additional permissions for cluster configuration
+
+You can add permissions to an Argo CD instance to manage cluster configuration. 
+
+.Procedure
+
+Create a ClusterRole with additional permissions and then create a new ClusterRoleBinding to associate the ClusterRole with a ServiceAccount:
+
+. Log in to the OpenShift Container Platform web console as an admin.
+. In the Openshift console, select **User Management** -> **Roles** -> **Create Role**. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
```

Comment:
```suggestion
. Log in to the {product-title} web console as an admin.
. In the web console, select **User Management** -> **Roles** -> **Create Role**. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
```

Reply:
OpenShift must always be preceded by "Red Hat" - using variables can help be consistent.